api/src/main/java/com/okta/idx/sdk/api/client/IDXClient.java

```diff
@@ -27,12 +27,14 @@
 import com.okta.idx.sdk.api.response.IDXResponse;
 import com.okta.idx.sdk.api.response.TokenResponse;
 
+import java.util.Optional;
+
 /**
  * Client to interact with the IDX backend APIs.
  */
 public interface IDXClient {
 
-    IDXClientContext interact() throws ProcessingException;
+    IDXClientContext interact(Optional<String> activationToken) throws ProcessingException;
```

Comment:
This is a binary API change. Should we do a new major version?

Reply:
yep, it'd be a major rev. I'll update the rc branch name.